charts/rancher-monitoring/v0.3.1/templates/crds.yaml

```diff
@@ -18,7 +18,9 @@ spec:
     plural: alertmanagers
   scope: Namespaced
   version: v1
+{{- end }}
 ---
```

Comment:
These should be included in the if blocks, or you'll get: 
```
---
---
---
```

Reply:
fixed.